<?xml version="1.0"?>
<launch>
    
    <arg name="namespace" default="bebop" />
    <arg name="ip" default="192.168.42.1" />
    <arg name="config_file" default="$(find bebop_driver)/config/defaults.yaml" />
    <arg name="camera_info_url" default="package://bebop_driver/data/bebop_camera_calib.yaml" />
    <group ns="$(arg namespace)">
        <node pkg="bebop_driver" name="bebop_driver" type="bebop_driver_node" output="screen">
            <param name="camera_info_url" value="$(arg camera_info_url)" />
            <param name="bebop_ip" value="$(arg ip)" />
            <rosparam command="load" file="$(arg config_file)" />
        </node>
        <include file="$(find bebop_description)/launch/description.launch" />
    </group>


    <group ns="bebop_driver">
        <node pkg="bebop_driver" name="obstacleDetect" type="obstacleDetect.py" output="screen" />
    </group>

    <group ns="face_detect">
        <node pkg="face_detect" name="facedetectServer" type="facedetectServer.py" output="screen" />
    </group>

    <group ns="bebop_driver">
        <node pkg="bebop_driver" name="collisionAvoid" type="collisionAvoid.py" output="screen" />
    </group>

    <group ns="bebop_driver">
        <node pkg="bebop_driver" name="bebopKeyTeleop" type="bebopKeyTeleop.py" output="screen" launch-prefix="xterm -e"/>
    </group>



</launch>
